README.md

```diff
@@ -214,7 +171,8 @@ Refer to the [Static Registration of measurements section]() for the detailed st
 
 To Statically register the examples provided, the user can copy the example directory with the service config file with the startup batch file, to the search paths and follow the [Setting up the Example Measurements](#setting-up-the-example-measurements) section to set up the measurements.
```

Comment:
This paragraph is worded awkwardly. I'd suggest
- Statically should not be capitalized.
- 'with the .serviceconfig file *and* the startup batch file
- No comma after 'the startup batch file'
- What does 'to the search paths' mean? Is that explained elsewhere?
- Doesn't this Setting up the Example Measurements link need to go to an external file now? We removed the section above.